src/taskpane/server.js

```diff
@@ -438,8 +541,14 @@ function getArtifacts(user, projectId, artifactTypeId, startRow, numberOfRows, a
       break;
     case ART_ENUMS.testSets:
       fullURL += "/test-sets?starting_row=" + startRow + "&number_of_rows=" + numberOfRows + "&sort_field=TestSetId&sort_direction=ASC&";
-      var rawResponse = fetcher(user, fullURL);
-      response = IS_GOOGLE ? JSON.parse(rawResponse) : rawResponse; // this particular return needs to be parsed here
+      if (IS_GOOGLE) {
+        var rawResponse = JSON.stringify(fetcher(user, fullURL));
+        response = IS_GOOGLE ? JSON.parse(rawResponse) : rawResponse; // this particular return needs to be parsed here

```

Comment:
And this line could be 
`response = JSON.parse(rawResponse);`

Reply:
fixed